.all-contributorsrc

```diff
@@ -455,6 +455,15 @@
         "bug",
         "userTesting"
       ]
+    },
+    {
+      "login": "dlemmermann",
+      "name": "Dirk Lemmermann",
+      "avatar_url": "https://avatars.githubusercontent.com/u/9534301?v=4",
+      "profile": "http://www.dlsc.com",
```

Comment:
"https" please

Reply:
I believe this information is taken directly from your GitHub profile by the AllContributors bot